src/main/java/it/pagopa/pm/gateway/dto/enums/PaymentRequestStatusEnum.java

```diff
@@ -0,0 +1,39 @@
+package it.pagopa.pm.gateway.dto.enums;
+
+import lombok.Getter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum PaymentRequestStatusEnum {
+
+    CREATED (1L, "Creata"),
+    AUTHORIZED (2L, "Autorizzazione concessa"),
+    DENIED (3L, "Autorizzazione negata"),
+    CANCELLED(4L, "Cancellata");
```

Comment:
is it correct to use enum status CANCELLED for refunded payment requests?
Maybe something like "REFUNDED" could be more appropriate: is this a requirement?

Reply:
As above, CANCELLED is correct.